tests/computed.test.tsx

```diff
@@ -64,7 +57,7 @@ describe('proxyWithComputed', () => {
   })
 
   it('computed getters and setters', async () => {
-    const computeDouble = jest.fn((x: number) => x * 2)
+    const computeDouble = vi.fn((x) => x * 2)
```

Comment:
How is `computeDouble` typed without `: number`?

Reply:
I assume it becomes `any`. Can we keep `: number`?

Reply:
Yes, I did it!